### DISCUSSION TITLE
How would you export utils/libs from a nuxt module server side ?

### DISCUSSION THREAD

**Discussion by theguriev** [nuxt/nuxt#19363] [Questions] [answered]
Hi guys, how do I get the auto-import to work for the server side?
For some reason this is working only for the client side 
<img width="541" alt="image" src="https://user-images.githubusercontent.com/8502021/222113472-216dcfb4-a4f3-4722-8791-004d2f7bd270.png">

It doesn't for the server side ( everything inside /server folder ). Should I add some extra argument or something to make it work on the server side?

**Comment by Baroshem** [comment, marked as answer]
Hey,

AFAIK the auto import works only for the vue related stuff as it is based on the auto import plugin by Antfu.

So you cannot have it for server side utilities

**Reply by Baroshem** [reply]
You can do something similar as I did for nuxt medusa module

https://github.com/Baroshem/nuxt-medusa/blob/main/src/module.ts Line 41

So create a virtual path to the server utility

And later on use it like this 

https://github.com/Baroshem/nuxt-medusa/blob/main/docs/content/2.features/2.serverMedusaClient.md

**Reply by theguriev** [reply]
Got it. Thank you ❤️

**Reply by theguriev** [reply]
Checked it out, and unfortunately, it doesn't work. 

<img width="573" alt="image" src="https://user-images.githubusercontent.com/8502021/222523257-80eeb9cb-c685-46a2-8cab-03437c39d628.png">


Here is the config file 
<img width="798" alt="image" src="https://user-images.githubusercontent.com/8502021/222523299-c3c022ce-7820-4645-a782-5f08d4acf7c2.png">

**Reply by Baroshem** [reply]
Could you explain what do you mean that it does not work? I see that you have installed medusa module in your appso this server utility should work easily. I will do my best to help :)

**Reply by theguriev** [reply]
Well, it can not resolve it for some reason.

**Reply by Baroshem** [reply]
Have you added a server:true in your configuration of the module?

**Reply by theguriev** [reply]
Yep, otherwise, the plugin `#medusa/server` would not be in the `alias`

**Reply by theguriev** [reply]
as a temporary workaround, I'll use something like 

`import { whatever } from '#imports' `

It's working perfectly.
